i18n/de/docusaurus-plugin-content-docs/current/contribution/30-overview/3-concepts/5-frontend-fragment.mdx

```diff
@@ -0,0 +1,111 @@
+Diese bieten grundsätzlich exakt dieselbe API wie die normalen [Flow Components](https://github.com/mittwald/flow/tree/main/packages/components), sind jedoch für die Verwendung in Remote DOMs (Frontend Fragmenten) ausgelegt.
+
+## Authentifizierung und Session Handling mittels Session-Token
+
+Für die Authentifizierung und das Session Handling im Frontend Fragment wird ein sogenannter Session-Token verwendet.
+Dieser Session-Token kann von einem Frontend Fragment jederzeit über Window Messages vom mStudio angefordert werden.
+Der Session-Token ist ein JWT, das Informationen über den aktuellen Nutzer und die Extension Instance enthält.
+Es stellt ein sogenanntes Online-Token dar, das nur 60 Sekunden gültig ist und somit nicht persistiert werden sollte.
+Das mStudio stellt sicher, dass ein Frontend Fragment nicht bei jeder Anfrage einen neuen Session-Token erstellt,
+sondern Session-Tokens innerhalb der ersten 40 Sekunden der Gültigkeit wiederverwendet werden.
+
+Session-Tokens enthalten stets eine für die jeweilige Session stabile Session ID.
+Diese Session ID kann genutzt werden, um die Session eines Nutzers zu identifizieren und bspw. Backend-Sessions abzubilden.
+Session-Cookies können nicht verwendet werden, da dies unter WebKit-basierten Browsern wie Safari nicht funktioniert.
+
+Ein Frontend Fragment muss bei Requests an das eigene Backend stets den Session-Token mitsenden.
+Das Session-Token enthält in den JWT-Claims unter dem Key `publicKeySerial` die Serial des Public Keys des mStudio, mit dem das Session-Token signiert wurde.
+Der Ed25519 Public Key kann über die mStudio API mithilfe der Operation <OperationLink tag="Marketplace" operation="extension-get-public-key" /> abgerufen werden.
+Dazu muss der `publicKeySerial` aus dem Session-Token als Serial und der Purpose `session_token` verwendet werden.
+Zur Vereinfachung der Prüfung der Signatur kann über den Query-Parameter `format` der Public Key im SPKI-Format, ASN.1 serialisiert und PEM kodiert, abgerufen werden.
+
+Das Backend muss den Session-Token bei jedem Request gegen den im Session-Token angegebenen Public Key des mStudio validieren, da der Session-Token das Authentifizierungsmerkmal des Nutzers darstellt.
+Dazu muss das JWT erst dekodiert werden, der Public Key über die API abgerufen und schließlich das JWT gegen den Public Key validiert werden.
+Der Public Key mit einem spezifischen Serial ist stabil und kann gecached/persistiert werden.
+
+Das Session-Token kann in Kombination mit einem Extension Secret für den Abruf eines Access Tokens verwendet werden.
+Dazu kann die Operation <OperationLink tag="Marketplace" operation="extension-authenticate-with-session-token" /> genutzt werden.
+Das dabei entstehende Access Token ist ebenfalls nur 60 Sekunden gültig und sollte nicht persistiert werden.
```

Comment:
Und was mache ich dann mit diesem Access-Token?

Reply:
ich habe in der Überarbeitung insgesamt deutlicher darauf hingewiesen wofür dass Access Token verwendet wird und würde hier an dieser Stelle nicht tiefer darauf eingehen.